### PR TITLE
DRILL-5937: ExecConstants: changed comment, timeout default is 30 sec.

### PR DESCRIPTION
**Pull request by pushpendra-jaiswal-90**
**Changed comment** to reflect default prepared statement timeout as 30 sec. (same as hive) .
10 seconds timeout is also failing in some cases in our mapr environment.

### CODE REVIEW

**Comment by pushpendra-jaiswal-90** [comment]
@vrozov @vdiravka Could you please review this?

**Comment by arina-ielchiieva** [comment]
@pushpendra-jaiswal-90 please mind Drill convention about naming commit message.
Example: https://github.com/apache/drill/commits/master

**Comment by pushpendra-jaiswal-90** [comment]
@arina-ielchiieva I've done the required changes.

**Comment by arina-ielchiieva** [comment]
@pushpendra-jaiswal-90 two more points:
1. In drill-module.conf default for prepare.statement.create_timeout_ms is set 10000 ms, which does not correspond to the value you have indicated. Please make sure both values are in sync.
2. Please do not do merge, it's not convenient when we are cherry-picking range of commits. But for your case I suggest you do squash the commits into one.

File: exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java [hunk 1]
```diff
@@ -558,7 +558,7 @@ private ExecConstants() {
 
   /**
    * Timeout for create prepare statement request. If the request exceeds this timeout, then request is timed out.
-   * Default value is 10mins.
+   * Default value is 30 seconds.
```
Comment (vrozov) [inline exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java]: Is it 30 seconds or 10 seconds? In any case, I don't think that the entire comment provides any additional info that cannot be inferred from the code itself.

**Comment by pushpendra-jaiswal-90** [comment]
@arina-ielchiieva Can you please tell me which drill-module.conf has it ?

File: exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java (hunk 1 above)
Comment (arina-ielchiieva) [reply exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java]: Agree, with Vlad, it better to remove this sentence at all: `Default value is 30 seconds.`

**Comment by arina-ielchiieva** [comment]
@pushpendra-jaiswal-90 try searching for the occurrence of `prepare.statement.create_timeout_ms` in the project using your IDE :)

File: exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java (hunk 1 above)
Comment (vrozov) [reply exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java]: I refer to the entire comment. "If the request exceeds this timeout, then request is timed out" does not add any value, it is simply a definition for a timeout. The same for `CREATE_PREPARE_STATEMENT_TIMEOUT_MILLIS` and "Timeout for create prepare statement request".

**Comment by pushpendra-jaiswal-90** [comment]
@arina-ielchiieva I have done the changes. Could you please review?

File: exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java [hunk 2]
```diff
@@ -557,8 +557,7 @@ private ExecConstants() {
   public static final LongValidator CODE_GEN_EXP_IN_METHOD_SIZE_VALIDATOR = new LongValidator(CODE_GEN_EXP_IN_METHOD_SIZE);
 
   /**
-   * Timeout for create prepare statement request. If the request exceeds this timeout, then request is timed out.
-   * Default value is 10mins.
+   * Default value is 30 seconds.
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java]: Maybe we should remove comment about the default as well? Actual default value can be found in drill-module.conf and if it changes, we won't need to modify it here.

**Comment by pushpendra-jaiswal-90** [comment]
@arina-ielchiieva I have removed comment.

**Comment by arina-ielchiieva** [comment]
+1

**Comment by pushpendra-jaiswal-90** [comment]
@vrozov  @vdiravka Could you please review this?

**Comment by vdiravka** [comment]
@pushpendra-jaiswal-90 
You already have +1 from Drill committer :) 
Your changes will be merged to master branch soon.

**Comment by vrozov** [comment]
@pushpendra-jaiswal-90 Please squash commits to a single commit.

**Pull request closed without merging**